ui/.eslintrc.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
     'react/jsx-uses-vars': [2],
     'react/jsx-no-undef': 'error',
     'no-console': 0,
-    'no-unused-vars': 'error',
+    'no-unused-vars': 'off',
```

Comment:
@ptdatta don't disable it globally, I wonder if you have reason to do so

Reply:
@ptdatta fix this.

Reply:
reverted this change 👍